src/components/send/SendHeader.js

```diff
@@ -162,7 +160,7 @@ export default function SendHeader({
             address: hexAddress,
             nickname: name,
             onDelete: () => {
-              onChangeAddressInput(contact?.ens);
+              onChangeAddressInput(contact?.ens ? contact?.ens : contact?.address);
```

Comment:
okay so basically this was triggering a `setToAddress` of `''` for non-ENS accounts, which threw us into a never ending loading state until you reselected the account you want to send to.